templates/customgroupfield.tpl

```diff
@@ -0,0 +1,18 @@
+{* FILE: customfieldnames/templates/customgroupfieldset.tpl to add custom field for custom data set fields *}
+<table id="custom-fields" class="form-layout">
```

Comment:
use "lcd-custom-fields" as the id; also fix in the js below

Reply:
Done